safe_cli/operators/hw_accounts/hw_account_manager.py

```diff
@@ -0,0 +1,140 @@
+from enum import Enum
+from typing import Dict, List, Optional, Set, Tuple
+
+from eth_typing import ChecksumAddress
+from prompt_toolkit import HTML, print_formatted_text
+
+from gnosis.eth.eip712 import eip712_encode
+from gnosis.safe import SafeTx
+
+from safe_cli.operators.hw_accounts.hw_account import HwAccount
+
+
+class HwWalletType(Enum):
+    TREZOR = "Trezor"
+    LEDGER = "Ledger"
+
+
+class HwAccountManager:
+    def __new__(cls):
+        if not hasattr(cls, "instance"):
+            cls.instance = super(HwAccountManager, cls).__new__(cls)
+        return cls.instance
+
+    def __init__(self):
+        self.accounts: Set[HwAccount] = set()
+        self.supported_hw_wallets: Dict[str, HwAccount] = {}
```

Comment:
```suggestion
        self.supported_hw_managers: Dict[str, HwAccount] = {}
```

Maybe it's more clear? Not a strong opinion here

Reply:
I'll review the names, because we have managers, wallets and accounts. Let me check better way to rename this classes.